app/javascript/src/components/Dashboard/Contacts/ContactList.jsx

```diff
@@ -0,0 +1,126 @@
+import React from "react";
+import { Checkbox, Avatar, Tooltip } from "neetoui";
+import deleteBtn from "images/DeleteButton";
+import editBtn from "images/EditButton";
+
+export default function ContactList({
+  selectedContactIds,
+  setSelectedContactIds,
+  setShowDeleteAlert,
+  contacts = [],
+}) {
+  const handleDelete = contactIds => {
+    setSelectedContactIds([contactIds]);
+    setShowDeleteAlert(true);
+  };
+
+  const deleteButton = contactId => {
+    return (
+      <Tooltip content="Delete Contact" position="bottom">
+        <div onClick={() => handleDelete(contactId)}>
+          <img src={deleteBtn} alt="deletebutton" />
+        </div>
+      </Tooltip>
+    );
+  };
+
+  const editButton = () => {
+    return (
+      <Tooltip className="mx-2" content="Edit Contact" position="bottom">
+        <div>
+          <img src={editBtn} alt="edit button" />
+        </div>
+      </Tooltip>
+    );
+  };
```

Comment:
No need to wrap UI components inside functions.

Reply:
Also please use neetoUI button with `style="icon"`.

Please refer the documentation for [Button](https://neeto-ui.netlify.app/#/button).